.github/workflows/kind.yml

```diff
@@ -277,6 +277,39 @@ jobs:
           path: log.tar.gz
           retention-days: 30
 
+  test-e2e-ipam-feature-enabled:
+    name: E2e tests on a Kind cluster on Linux with FlexibleIPAM feature enabled
+    needs: [build-antrea-coverage-image]
+    runs-on: [ubuntu-latest-4-cores]
+    steps:
+      - name: Free disk space
```

Comment:
I would remove this step, unless you know that there is a shortage of disk space on these runners